app-install-banner/deferred-banner/index.html

```diff
@@ -42,16 +42,18 @@ <h1>Web App Install Banner: Defer prompt of banner sample</h1>
        This API and example shows you how you can defer the prompt until a later time, in this case when the 
        user clicks the button below.  If the button below is disabled, Chrome has chosen now is not a good time to 
        show the prompt.</p>
+
+    <p>If the button is disabled, come back in a few minutes and the button will be enabled.</p>
+
+    <p>For testing, we encourage you to force the button to be enabled by setting the
```

Comment:
We may want to wait a bit before updating this sample as @dmurph is removing the engagement requirement for the `beforeinstallprompt` event in Chrome 131